### ISSUE TITLE
fotex.breaks package loading

### ISSUE THREAD

**Issue by keinstein** [sebastianrahtz/passivetex#1]
Hi,
the file fotex.sty contains the code 

`\let\FOinputIfFileExists\InputIfFileExists
\def\InputIfFileExists#1#2#3{%
 {\def\@tempa{#1}\ifx\@tempa\supppdf\else
   \FOinputIfFileExists{#1}{#2}{#3}\fi}}'

The duplicate braces of the new InputIfFileExists break package loading.

If I remove  the grouping braces and leave only the defining braces I get different erros which might occur due to bad parsing of a self-closed fo:block tag.

**Comment by jamescummings** [comment]
Given Sebastian Rahtz has sadly passed away he is unlikely to respond to any issues on this repository. I'd suggest following up with the TeX community to see if there are those wishing to maintain passivetex who would then fork this repository.

**Comment by keinstein** [comment]
@jamescummings I realized that some hours after posting the issue. But, thank you for the information, anyway.

Unfortunately I don't know a better place to share my thoughts, at the moment. I will even create a pull request, when I get a solution that works for me at the moment. I can't send the patch directly to CTAN as I cannot run the test suite and I'm not familiar with neither xmltex, nor passivetex. I can't even run the test-suite as it depends on several files on Sebastians harddisk. Some can be found in some small corner in the internet. Some others were available at http://www.tei-c.org/Stylesheets/. From these files I found so far https://web.archive.org/web/20030901000000*/http://www.tei-c.org.uk/Stylesheets/tei.xsl from 2003. Unfortunately this file depends on other files which are not available on archive.org. So I can't even run the test suite.

**Comment by jamescummings** [comment]
Hi @keinstein,

The TEI repositories are all available here on github.  See http://github.com/TEIC/  (for example the Stylesheets repository).  I'm not very familiar with passive tex but if it comes down to a problem with the TEI Stylesheets then do put an issue in that repository.

-James

**Comment by keinstein** [comment]
@jamescummings Yes, I've found them, but they do not contain the mentioned xsl files, which should be in the root directory of the Stylesheets repository (as you can deduce from the archive URL). But there are no .xsl files. I'm not shure that it is an issue with the Sytelesheets repository. For me it looks like these files have been part of TEIs WWW presentation and not of the Stylesheets distribution.

Maybe they should be available as part of the documentation, but so far I did not find these particular xml files that depend on the XSL files in the Stylesheets repository. I will think over it a little bit…

Anyways, thanks for the input.
